openquake/hazardlib/geo/surface/gridded.py

```diff
@@ -133,10 +133,13 @@ def get_top_edge_depth(self):
      
     def get_strike(self):
         """
-        Compute surface's strike as decimal degrees in a range ``[0, 360)``.
-        The actual definition of the strike might depend on surface geometry.
+        params:
```

Comment:
As I wrote in my previous comment, pls remove the description of param since this method does not ask for input parameters

Reply:
okay